EnvironmentSimulator/Modules/RoadManager/RoadManager.cpp

```diff
@@ -3777,6 +4520,146 @@ void RMObject::SetRepeat(Repeat* repeat)
 	repeat_ = repeat;
 }
 
+void RMObject::Save(pugi::xml_node& objects)
+{
+	auto object = objects.append_child("object");
+	object.append_attribute("type").set_value(type_.c_str());
+	//object.append_attribute("subtype").set_value() TODO:
+	//object.append_attribute("dynamic").set_value() TODO:
+	object.append_attribute("name").set_value(name_.c_str());
+	object.append_attribute("id").set_value(id_);
+	object.append_attribute("s").set_value(s_);
+	object.append_attribute("t").set_value(t_);
+	object.append_attribute("zOffset").set_value(z_offset_);
+	//object.append_attribute("validLength").set_value(); TODO:
```

Comment:
remove

Reply:
These kind of commented out stuff I left since there are no parser for these attributes. Will add TODO mark aswell.